lib/dkim.js

```diff
@@ -498,6 +501,24 @@ function DKIMVerify(email, callback, sigs) {
     });
 }
 
+function checkMissingSigningHeaders(sig) {
+  var headersToWarnOnMissing = ['date', 'subject'],
+    lowerCaseHeaders = [],
+    i;
+
+  //convert all headers to lowercase
+  for(i = 0; i < sig.h.length; i++){
+    lowerCaseHeaders.push(sig.h[i].toLowerCase());
+  }
+
+  for (i = 0; i < headersToWarnOnMissing.length; i++) {
+    if (lowerCaseHeaders.indexOf(headersToWarnOnMissing[i]) === -1) {
+      sig.warnings.push('Signing message ' + headersToWarnOnMissing[i] + ' is strongly recommended!');
```

Comment:
no objection as i was also thinking `message` might be redundant or unnecessary.